### PR TITLE
Add terms of use page

### PR DESCRIPTION
**Pull request by NingW101**
links in sub page are WIP

### CODE REVIEW

File: samples/phi3-webgpu/index.html [hunk 1]
```diff
@@ -5,9 +5,53 @@
...
+        margin: 0 auto;
+        text-align: center;
+        border-top: 1px solid transparent;
+      }
+    </style>
   </head>
   <body>
     <div id="root"></div>
+    <div class="divider"></div>
+    <footer>
```
Comment (ibelem) [inline samples/phi3-webgpu/index.html:47]: @NingW101 Do we have common footer for all demos? If so I prefer to add a <footer></footer> tag in each page then insert the nodes by using common script e.g. https://github.com/webmachinelearning/webnn-samples/blob/master/common/component/component.js#L580 to save the maintenance effort. What do you think?

**Review by ibelem — CHANGES_REQUESTED** [review]

File: samples/phi3-webgpu/index.html (hunk 1 above)
Comment (NingW101) [reply samples/phi3-webgpu/index.html:47]: Yes, good suggestion, currently we do not have common footer, I'll make footer template and replace them

**Comment by NingW101** [comment]
Change status to `Ready for review` as the naming reviewer is fine for the `Web AI Showcase`

**Review by ibelem — CHANGES_REQUESTED** [review]

**Review by ibelem — APPROVED** [review]
LGTM

**Pull request merged**